### PR TITLE
Added actions menu to read inspection screen

### PR DESCRIPTION
**Pull request by BenHakimIlyass**

### CODE REVIEW

File: src/screens/InspectionRead/index.jsx [hunk 1]
```diff
@@ -104,6 +108,57 @@ const styles = StyleSheet.create({
...
+    setMenuOpen(true);
+  }, []);
+
+  const handleDismissMenu = useCallback(() => {
+    setMenuOpen(false);
+  }, []);
+
+  const handleGoToEditInspection = useCallback(() => {
+    navigation.navigate(INSPECTION_UPDATE);
+  }, [navigation]);
```
Comment (natsukiai) [inline src/screens/InspectionRead/index.jsx:127]: Can we make a hook of it ?

**Review by natsukiai — APPROVED** [review]

**Comment by franklinkemta** [comment]
Hello i want to merge your PR to re-use the menu.
Also can you update the menu so when an option is selected the menu pops out ?

**Comment by BenHakimIlyass** [comment]
@franklinkemta I will add the auto close functionality, once I finish my current task.

**Comment by franklinkemta** [comment]
> @franklinkemta I will add the auto close functionality, once I finish my current task.

My thanks !

**Pull request merged**